wakuv2/nwaku_test.go

```diff
@@ -545,6 +545,44 @@ func TestPeerExchange(t *testing.T) {
 	require.NoError(t, discV5Node.Stop()) */
 }
 
+func TestDnsDiscover(t *testing.T) {
+	logger, err := zap.NewDevelopment()
+	require.NoError(t, err)
+
+	nodeConfig := Config{
+		UseThrottledPublish: true,
+		ClusterID:           16,
+	}
+
+	// start node that will initiate t
+	nodeWakuConfig := WakuConfig{
+		EnableRelay:   true,
+		LogLevel:      "DEBUG",
+		ClusterID:     16,
+		Shards:        []uint16{64},
+		Discv5UdpPort: 9040,
+		TcpPort:       60040,
+	}
+
+	node, err := New(nil, "", &nodeConfig, &nodeWakuConfig, logger.Named("node"), nil, nil, nil, nil)
+	require.NoError(t, err)
+	require.NoError(t, node.Start())
+
+	time.Sleep(1 * time.Second)
+
+	sampleEnrTree := "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.prod.status.nodes.status.im"
+	nameserver := "1.1.1.1"
```

Comment:
same here

Reply:
In the future we may need to expose, through `libwaku`, the nwaku's used DNSs.
Something to be considered once the following is completed: https://github.com/waku-org/nwaku/issues/1490